zmesh.pyx

```diff
@@ -57,9 +76,22 @@ class Mesher:
 
```

Comment:
I think we are using simplification error of 40, not 8?
default simplification_factor should be higher too?